runtime_scan/pkg/orchestrator/assetscanprocessor/vulnerabilities.go

```diff
@@ -63,9 +63,9 @@ func (srp *ScanResultProcessor) reconcileResultVulnerabilitiesToFindings(ctx con
 	logger.Infof("Found %d existing vulnerabilities findings for this scan", len(existingMap))
 	logger.Debugf("Existing vulnerabilities map: %v", existingMap)
 
-	if scanResult.Vulnerabilities != nil && scanResult.Vulnerabilities.Vulnerabilities != nil {
+	if assetScan.Vulnerabilities != nil && assetScan.Vulnerabilities.Vulnerabilities != nil {
 		// Create new findings for all the found vulnerabilities
-		for _, vuln := range *scanResult.Vulnerabilities.Vulnerabilities {
+		for _, vuln := range *assetScan.Vulnerabilities.Vulnerabilities {
```

Comment:
Just a note for future here, `assetScan.Vulnerabilities.Vulnerabilities` seems cumbersome, would be good to replace it with something more straightforward.